templates/program/nph/order/sample-finalize.html.twig

```diff
@@ -222,10 +221,10 @@
                                                         Please verify the unit of measurement is correct. (For reference 1mL = 1000uL)'
                                                     </span>
                                                 </td>
-                                                <td>
-                                                    {{ aliquot['units'] }}
+                                                <td class="aliquot volume nopadleft" style="position: relative">
+                                                    <span style="position: absolute; bottom: 7px; left: 0;">{{ aliquot['units'] }}</span>
```

Comment:
You need to apply the same styles here as well.

![Screen Shot 2023-02-15 at 3 45 26 PM](https://user-images.githubusercontent.com/6041980/219175565-00bf8617-d6cf-4bc2-8785-1320c40f27eb.png)